Content.Server/VoiceMask/VoiceMaskComponent.cs

```diff
@@ -6,4 +8,14 @@ public sealed partial class VoiceMaskComponent : Component
     [ViewVariables(VVAccess.ReadWrite)] public bool Enabled = true;
 
     [ViewVariables(VVAccess.ReadWrite)] public string VoiceName = "Unknown";
+
+    /// <summary>
+    /// If EnableSpeechVerbModification is true, overrides the speech verb used when this entity speaks.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [ValidatePrototypeId<SpeechVerbPrototype>]
+    public string SpeechVerb = "Default";
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public bool EnableSpeechVerbModification = false;
 }
```

Comment:
Why aren't these datafields?

Reply:
Huh, good point. They probably all should be, right?